Fix/fix typos login setup (#18511)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

---
title: Enable YugabyteDB Anywhere authentication via LDAP
headerTitle: Enable YugabyteDB Anywhere authentication via LDAP

> description: Use LDAP to enable log in to YugabyteDB Anywhere.

Reviewer: ```suggestion
description: Use LDAP for authentication in YugabyteDB Anywhere.
```
